src/main/java/org/jabref/JabRefGUI.java

```diff
@@ -89,8 +87,7 @@ private void openWindow(Stage mainStage) {
                 event.consume();
             }
         });
-
-        executorService.execute(this::openDatabases);
+        this.openDatabases();
```

Comment:
This should still run later on the javafx (Platform.runLater)

Reply:
But why? It's already on the FX thread or do you mean it should be quueed?

Reply:
Yes, it should happen after the UI is created so that the user get's a quicker feedback that JabRef started.